nixos/modules/services/web-servers/keter/default.nix

```diff
@@ -0,0 +1,105 @@
+# https://nixos.wiki/wiki/Module
```

Comment:
Comment seems redundant. Also you should run `nixpkgs-fmt` on this file too.

Reply:
I think I formatted all the files

Reply:
Comment still here